dev-util/klee/klee-9999.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils llvm
+
+DESCRIPTION="Symbolic virtual machine built on top of the LLVM compiler infrastructure"
+HOMEPAGE="http://klee.github.io/"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/klee/klee.git"
+else
+	SRC_URI="https://github.com/klee/${PN}/archive/v1.4.0.tar.gz -> ${P}.tar.gz"
```

Comment:
Seems like you forgot to include the ebuild for 1.4.0 and remove this hardcoded value :)

Reply:
Fixed SRC_URI, but there is no buildable release at the moment since 1.4.0 has an `#include <llvm/Support/TimeValue.h>` line and `TimeValue.h` was removed in LLVM 3.8.

Reply:
Then we should create a snapshot release.

Reply:
Added a snapshot of the current master.

Reply:
Looks good at first glance